firmware/klipper_configurations/Octopus/Voron2_Octopus_Config.cfg

```diff
@@ -0,0 +1,646 @@
+step_pin: PF11
+dir_pin: PG3
+enable_pin: !PG5
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+endstop_pin: PG10
+##  Z-position of nozzle (in mm) to z-endstop trigger point relative to print surface (Z0)
+##  (+) value = endstop above Z0, (-) value = endstop below
+##	Increasing position_endstop brings nozzle closer to the bed
+##  After you run Z_ENDSTOP_CALIBRATE, position_endstop will be stored at the very end of your config
+#position_endstop: -0.5
+##--------------------------------------------------------------------
+
+##	Uncomment below for 250mm build
+#position_max: 240
+
+##	Uncomment below for 300mm build
+#position_max: 290
+
+##	Uncomment below for 350mm build
+#position_max: 340
+
+##--------------------------------------------------------------------
+position_min: -5
+homing_speed: 8
+second_homing_speed: 3
+homing_retract_dist: 3
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z]
+uart_pin: PC6
+interpolate: true
+run_current: 0.8
+hold_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##	Z1 Stepper - Rear Left
+##  Connected to MOTOR_3
+[stepper_z1]
+step_pin: PG4
+dir_pin: !PC1
+enable_pin: !PA0
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z1]
+uart_pin: PC7
+interpolate: true
+run_current: 0.8
+hold_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##	Z2 Stepper - Rear Right
+##  Connected to MOTOR_4
+[stepper_z2]
+step_pin: PF9
+dir_pin: PF10
+enable_pin: !PG2
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z2]
+uart_pin: PF2
+interpolate: true
+run_current: 0.8
+hold_current: 0.80
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##	Z3 Stepper - Front Right
+##  Connected to MOTOR_5
+[stepper_z3]
+step_pin: PC13
+dir_pin: !PF0
+enable_pin: !PF1
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z3]
+uart_pin: PE4
+interpolate: true
+run_current: 0.8
+hold_current: 0.80
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+
+#####################################################################
+# 	Extruder
+#####################################################################
+
+#	Connected to MOTOR_0
+#   Heater - HE0
+#   Thermistor - T0
+[extruder]
+step_pin: PE2
+dir_pin: PE3
+enable_pin: !PD4
+##	Update value below when you perform extruder calibration
+##	If you ask for 100mm of filament, but in reality it is 98mm:
+##	rotation_distance = <previous_rotation_distance> * <actual_extrude_distance> / 100
+##  22.6789511 is a good starting point
+rotation_distance: 22.6789511	#Bondtech 5mm Drive Gears
+##	Update Gear Ratio depending on your Extruder Type
+##	Use 50:17 for Afterburner/Clockwork (BMG Gear Ratio)
+##	Use 80:20 for M4, M3.1
```

Comment:
Add LGX rotation distance: 8 according to BMG, 7.780624 according to my config (hot end removed)